system/metrics/sinks/prometheus/prometheusSink.cpp

```diff
@@ -240,16 +243,16 @@ const std::vector<std::string> &PrometheusMetricSink::getHistogramLabels(const s
 
     // Add the inf label
     std::string infLabel(name);
-    infLabel.append("_bucket{\"le=+Inf\"}");
+    infLabel.append("_bucket{le=\"+Inf\"}");
     labels.emplace_back(infLabel);
 
     // Sum and count
-    std::string sumLabel;
-    sumLabel.append("_sum");
+    std::string sumLabel(name);
+    sumLabel.append("_sum{}");
     labels.emplace_back(sumLabel);
 
-    std::string countLabel;
-    countLabel.append("_count");
+    std::string countLabel(name);
+    countLabel.append("_count{}");
```

Comment:
Curious why the '{}" is present. Braces are not part of what is shown on the Prometheus page documenting their exposition formats.  See https://prometheus.io/docs/instrumenting/exposition_formats/

Please double check the reason for "{}"

Reply:
Great observation, it's legal syntax as confirmed by promtool and prometheous graph UI used as a label container. It's optional, and honestly it was added as I tested why your histograms were deemed invalid.

Reply:
What stands out to me is the fact there's currently no mechanism (at least not one I could see) to populate the histogram buckets w/ any available metadata as labels. That will likely need to be fixed.